lib/features/email/domain/usecases/get_email_content_interactor.dart

```diff
@@ -28,6 +31,14 @@ class GetEmailContentInteractor {
   ) async* {
     try {
       yield Right<Failure, Success>(GetEmailContentLoading());
+
+      final isDetailEmailCacheExisted = await _openedEmailCacheManager.isOpenedDetailEmailCached(accountId, session.username, emailId);
```

Comment:
IMO, no need to check it.
Get it directly:
- if not null -> yield and return
- if null -> reuqest to network

IMO, it will reduce the request to cache, better in response